GOVERNANCE.md

```diff
@@ -0,0 +1,60 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="ROCm governance model">
+  <meta name="keywords" content="ROCm, governance">
+</head>
+
+# Governance model
+
+ROCm is a software stack made up of a collection of drivers, development tools, and APIs that enable
+GPU programming from the low-level kernel to end-user applications.
+
+Components of ROCm that are inherited from external projects (such as
+[LLVM](https://github.com/RadeonOpenCompute/llvm-project) and
+[Kernel driver](https://github.com/RadeonOpenCompute/ROCK-Kernel-Driver)) follow their own
+governance model and code of conduct. All other components of ROCm are governed by this
+document.
+
+## Governance
+
+ROCm is led and managed by AMD.
```

Comment:
legal name of AMD?

Reply:
I don't have super strong feelings (tend to prefer 'AMD')...is there a reason (legal or otherwise) for having the long name?

Reply:
Sorry, I reverted this back to just AMD.  It sounds better to me.